### PR TITLE
Proofreading pass

### PR DESCRIPTION
**Pull request by CactusPuppy**
This pull request focuses primarily on the Privacy Policy page, with minor adjustments to the Terms of Service and Discord banner partial. A small adjustment to the `.gitignore` file was also introduced to facilitate the use of portable gem sets instead of needing to utilize the Ruby installation's gem installation path to prevent dependency conflicts and pollution.

### CODE REVIEW

File: app/views/application/banners/_elo_hell_discord.html.erb [hunk 1]
```diff
@@ -1,9 +1,9 @@
 <div class="well well--dark <%= current_user.present? ? "mt-1/4" : "mt-1/1" %>">
 
   <div class="mb-1/4 lg:mb-0 mr-1/4">
-    <h3 class="mt-0 mb-0"><strong>Elo Hell Workshop Discord</strong></h3>
+    <h3 class="mt-0 mb-0"><strong>Elo Hell Workshops Discord</strong></h3>
```
Comment (Zezombye) [inline app/views/application/banners/_elo_hell_discord.html.erb:4]: there's only one workshop wtf

File: app/views/pages/privacy_policy.html.erb [hunk 2]
```diff
@@ -1,49 +1,49 @@
...
+<h2 class="mt-1/1" id="data-we-collect">Data We Collect and How We Use It</h2>
 
-<p>When you use the website you are given a token to identify you. This token is completely annonymous and can not be used to identify you personally and expires after 8 hours of no activity. This token is saved in our database and is kept for a maximum of 1 month. An annoymized IP address is saved along with this token. Instead of saving your full IP address the last digits are removed. For example, if your IP address is 192.169.123.45 your IP address is saved as 192.169.123.0.<br></p>
+<p>When you visit Workshop.codes you are given a token (in the form of a browser cookie) to identify you (hereafter referred to as the visitor token). The visitor token is completely anonymous, cannot be used to identify you personally, and expires after 8 hours of no activity. Visitor tokens are saved in our database for a maximum of 1 month. An anonymized IP address is saved along with this token. We anonymize IP addresses by removing the last octet (set of digits after the last period) in the address. For example, if your IP address was 192.169.123.45, your IP address would be saved as 192.169.123.0.<br></p>
```
Comment (Zezombye) [inline app/views/pages/privacy_policy.html.erb:24]: How does this work for ipv6?

Comment (Mitcheljager) [reply app/views/pages/privacy_policy.html.erb:24]: We use [Ahoy](https://github.com/ankane/ahoy);

> Ahoy can mask IPs with the same approach Google Analytics uses for IP anonymization. This means:
>      For IPv4, the last octet is set to 0 (8.8.4.4 becomes 8.8.4.0)
>      For IPv6, the last 80 bits are set to zeros (2001:4860:4860:0:0:0:0:8844 becomes 2001:4860:4860::)

Comment (CactusPuppy) [reply app/views/pages/privacy_policy.html.erb:24]: I could probably just link to Ahoy for that for future-proofing I suppose

File: app/views/application/banners/_elo_hell_discord.html.erb (hunk 1 above)
Comment (CactusPuppy) [reply app/views/application/banners/_elo_hell_discord.html.erb:4]: I didn't name our Discord >:(

**Review by Mitcheljager — APPROVED** [review]

**Pull request merged**